particle-iot/particle-cli
# feature/v2

## Description

Adding official Node v12 support, dropping Node v6 support (hence the major bump)


## How to Test

Install and run Node v12 - fwiw, here's what i do:

```
>>brew install node@12
>>brew unlink node@8 && brew link node@12 --overwrite --force
```

...and verify your versions:

```
>>node -v
v12.13.1
>>npm -v
6.12.1
```

...then reinstall local dependencies

```
>>npm run reinstall
```

...and confirm _no changes_ are made to `npm-shrinkwrap.json`

With that all in place, connect your device and [run the end-to-end tests](https://github.com/particle-iot/particle-cli/blob/feature/v2/test/README.md)

```
>>npm test
```

Lastly, broadly try commands. 

## Related Issues / Discussions

https://github.com/particle-iot/particle-cli/issues/508
https://github.com/particle-iot/particle-cli/pull/512


## Completeness
